t/app/controller/waste_peterborough_bulky.t

```diff
@@ -64,6 +66,7 @@ create_contact(
     { code => 'payment' },
     { code => 'payment_method' },
     { code => 'property_id' },
+    { code => 'bulky_text_updates' },
```

Comment:
This is probably me not paying close enough attention but it took me ages of wondering why the feature wasn't working before I spotted this line and realised I also needed to add a `bulky_text_updates` hidden field to my existing `Bulky collection` category. Always helpful to please include notes for the reviewer when adding code that needs configuration/changes beyond checking out the branch.